main.go

```diff
@@ -84,6 +86,9 @@ func init() { //nolint:gochecknoinits //reason this way we ensure schemes are al
 // +kubebuilder:rbac:groups=authorino.kuadrant.io,resources=authconfigs,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=datasciencecluster.opendatahub.io,resources=datascienceclusters,verbs=get;list;watch
 // +kubebuilder:rbac:groups=dscinitialization.opendatahub.io,resources=dscinitializations,verbs=get;list;watch
+// +kubebuilder:rbac:groups=nim.opendatahub.io,resources=accounts,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=nim.opendatahub.io,resources=accounts/status,verbs=get;update;patch
+// +kubebuilder:rbac:groups=nim.opendatahub.io,resources=accounts/finalizers,verbs=update
```

Comment:
Does the webhook require anything more than _get;list;watch_ for account resources?
I don't think the webhook needs the status, and we don't have a finalizer.

Reply:
These permissions are what a controller typically has for a new API.
I was supposed to add these in the PR for the API.

Reply:
In general, I think these should be added with the controller, not the API.
Regardless, these are more than what we need. For the accounts, we only need _get_, _list_, and _watch_, which already exist in the controller PR. I mistakenly left out the status permissions, but this was fixed. As for the finalizer permissions, we don't use finalizers.